sdk/service/backend.go

```diff
@@ -53,6 +53,9 @@ func (s *Backend) handler(request cosmostypes.Request, msg cosmostypes.Msg) cosm
 	case msgCreateService:
 		srv, err := s.Create(request, &msg)
 		if err != nil {
+			if cerr, ok := err.(cosmostypes.Error); ok {
+				return cerr.Result()
+			}
```

Comment:
that's a good idea but can you move it to a new handler wrapper in `cosmos/module.go` (like the querier wrapper)?

Reply:
I'm not sure if we need to implement that into handler wrapper.

Querier expects response (bytes) and error. In querier wrapper a response is marsla from interface into bytes.

For Handler there is no response expected, only `result` interface. So we don't need any wrapper there.